ssl/s3_lib.c

```diff
@@ -842,8 +842,71 @@ static SSL_CIPHER ssl3_ciphers[] = {
      SSL_AES128GCM,
      SSL_AEAD,
      TLS1_3_VERSION, TLS1_3_VERSION,
+     SSL_kANY,
+     SSL_aANY,
+     SSL_HIGH,
+     SSL_HANDSHAKE_MAC_SHA256 | TLS1_PRF_SHA256,
```

Comment:
Yeah, I think they're always the same in TLS1.3, so there doesn't seem any reason to have two flags which are always going to indicate the same hash.